src/setupTest.js

```diff
@@ -18,6 +18,7 @@ process.env.PUBLISHER_BASE_URL = 'http://localhost:18400';
 process.env.ECOMMERCE_BASE_URL = 'http://localhost:18130';
 process.env.LANGUAGE_PREFERENCE_COOKIE_NAME = 'openedx-language-preference';
 process.env.LMS_BASE_URL = 'http://localhost:18000';
+process.env.LOGIN_URL = 'http://localhost:2000';
```

Comment:
Watch out! This says `LOGIN_URL` still. 😀

Reply:
Whoops!!! 🤭